python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -673,3 +673,16 @@ def f(x):
     data.apply(f)
 
     assert precompiled.currsize == 1
+
+
+@pytest.mark.parametrize(
+    "data", [[1.0, 0.0, 1.5], [1, 1, 2], [True, False, True]]
```

Comment:
Maybe add a falsy integer.
```suggestion
    "data", [[1.0, 0.0, 1.5], [1, 0, 2], [True, False, True]]
```